specification/subscription/resource-manager/Microsoft.Subscription/preview/2024-08-01-preview/subscriptions.json

```diff
@@ -753,6 +753,144 @@
         }
       }
+    },
+    "/providers/Microsoft.Subscription/transfers/{transferId}": {
+      "put": {
+        "description": "Transfer subscription billing ownership to destination email id. TransferId is for tracking purposes only and is different from the system-generated TransferId",
+        "operationId": "Transfers_SendExport",
+        "x-ms-examples": {
+          "CreateTransfer": {
+            "$ref": "./examples/createTransfer.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Transfer creation request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TransferCreationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer Details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "description": "Retrieves transfer details using a system-generated TransferId. ",
+        "operationId": "Transfers_GetById",
+        "x-ms-examples": {
+          "GetTransferById": {
+            "$ref": "./examples/getTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "description": "Cancel the transfer by using its system generated TransferId.",
+        "operationId": "Transfers_CancelById",
+        "x-ms-examples": {
+          "GetTransferById": {
+            "$ref": "./examples/cancelTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
```

Comment:
>TransferDetails [](http://example.com/codeflow?start=37&length=15)

Against RPC to have a response body for a delete API. Consider the POST actions suggested ?

Reply:
@abdulgagit Please schedule an ARM API modeling office hours to discuss on the choice of this API as mentioned in the other comment.
It seems to cancel an operation and not delete a resource as such. 

In general for a Delete API: Please check [https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/delete-resource.md](https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/delete-resource.md) for the 200 vs 204 differences.

Reply:
resolving this comment as it addressed 204